6/ansible-6.build

```diff
@@ -48,30 +48,30 @@ community.routeros: >=2.0.0,<3.0.0
 community.sap: >=1.0.0,<2.0.0
 community.skydive: >=1.0.0,<2.0.0
 community.sops: >=1.2.0,<2.0.0
-community.vmware: >=2.2.0,<3.0.0
+community.vmware: >=2.3.0,<3.0.0
 community.windows: >=1.9.0,<2.0.0
-community.zabbix: >=1.5.0,<2.0.0
+community.zabbix: >=1.6.0,<2.0.0
 containers.podman: >=1.9.0,<2.0.0
 cyberark.conjur: >=1.1.0,<2.0.0
 cyberark.pas: >=1.0.0,<2.0.0
 dellemc.enterprise_sonic: >=1.1.0,<2.0.0
-dellemc.openmanage: >=5.2.0,<6.0.0
+dellemc.openmanage: >=5.3.0,<6.0.0
 dellemc.os10: >=1.1.0,<2.0.0
 dellemc.os6: >=1.0.0,<2.0.0
 dellemc.os9: >=1.0.0,<2.0.0
-f5networks.f5_modules: >=1.15.0,<2.0.0
+f5networks.f5_modules: >=1.16.0,<2.0.0
 fortinet.fortimanager: >=2.1.0,<3.0.0
 fortinet.fortios: >=2.1.0,<3.0.0
 frr.frr: >=1.0.0,<2.0.0
```

Comment:
```
[WARNING]: Collection frr.frr does not support Ansible version 2.13.0rc1
```

Reply:
There is a version 1.0.4 of frr.frr planned to be out soon with a fixed ansible requirement.

Reply:
The new version didn't make it out in time for the rebuild of 6.0.0a2. It should be in the next alpha release.